Sources/Apollo/GraphQLOperation.swift

```diff
@@ -1,10 +1,19 @@
+/// Enumeration of the possible GraphQL operations that can be executed against an API endpoint.
 public enum GraphQLOperationType {
   case query
   case mutation
   case subscription
+
+  var cacheKey: String {
+    switch self {
+    case .query: return "QUERY_ROOT"
+    case .mutation: return "MUTATION_ROOT"
+    case .subscription: return "SUBSCRIPTION_ROOT"
+    }
+  }
 }
 
-public protocol GraphQLOperation: class {
+public protocol GraphQLOperation: AnyObject {
```

Comment:
Why this change?

Reply:
I was in this file at some previous iteration, but here's the rationale: `protocol P: class` is considered deprecated without a warning

https://forums.swift.org/t/class-only-protocols-class-vs-anyobject/11507/4

Reply:
Hm...I'll ping Slava about that on twitter, that post is from 2018 and I wonder why, if that's the case, they haven't added the warning

Reply:
Sayeth the oracle: [Still the same](https://twitter.com/slava_pestov/status/1326976867627380744)